source/views/streaming/streams/list.tsx

```diff
@@ -28,6 +29,22 @@ const groupStreams = (entries: StreamType[]) => {
 	return toPairs(grouped).map(([title, data]) => ({title, data}))
 }
 
+const getEnabledCategories = (filters: ListType[]) => {
```

Comment:
Any particular reason to use a `const` `=>` function here?

Reply:
Borrowing from Hawken's earlier answer in this comment https://github.com/StoDevX/AAO-React-Native/pull/6412#discussion_r979440572. I see it as helpful in avoiding pitfalls when dealing with `this`and caller context.